client/components/chart/index.jsx

```diff
@@ -120,9 +128,19 @@ class Chart extends React.Component {
 					barClick={ barClick }
 					data={ data }
 					yAxisMax={ yMax }
-					chartWidth={ width }
 					isTouch={ hasTouch() }
+					chartWidth={ width }
+					setTooltip={ this.setTooltip }
 				/>
+				<Tooltip
```

Comment:
Use `this.state.tooltipVisible && <Tooltip ... />` to prevent creating unneeded tooltip JSX elements.